modules/api/php/projectimagesrow.class.inc

```diff
@@ -33,7 +44,18 @@ class ProjectImagesRow implements \LORIS\Data\DataInstance
      */
     public function __construct(array $row)
     {
-        $this->DBRow = $row;
+        $this->_candid     = $row['Candidate'] ?? null;
```

Comment:
Could maybe use the CandID object here.

Reply:
This is coming from the database and I don't think instantiating a CandID is useful here.